frontend/src/store/modules/auth/mutations.js

```diff
@@ -10,15 +12,18 @@ export default {
     },
     [USER_LOGOUT]: (state) => {
         state.token = undefined;
-        state.currentUser = {
-            name: '',
-            email: '',
-            id: null
-        };
+        state.currentUser = EMPTY_USER();
```

Comment:
probably better just define `currentUser` as `undefined` (in state object definition too) instead of define separate function for generating empty user